### ISSUE TITLE
Build the PMUv3 plugin

### ISSUE THREAD

**Issue by nehcauhiy** [ArmDeveloperEcosystem/arm-learning-paths#1258]
**Type of issue**
- [ ] Instructions in the learning path do not work.
- [ ] Spelling error in the learning path.
- [*] Instructions in the learning path are hard to follow.
- [ ] Instructions to contribute a new learning path are not clear.
- [ ] Other

I am trying to build the PMUv3 plugin, but the source code of PMUv3 plugin is not found.

Thanks,
Hua

**Comment by jasonrandrews** [comment]
@nehcauhiy 

Sorry, there is currently an issue with the source code, and it is being fixed. It's not accessible at this time, but hopefully access will be restored soon.

Thanks for reporting it.

**Comment by jasonrandrews** [comment]
Hi @nehcauhiy 
The new repository is now available, you can clone using:

git clone https://github.com/ARM-software/PMUv3_plugin.git

The Learning Path is fixed and will be visible on learn.arm.com on the next update, later this week.

**Comment by nehcauhiy** [comment]
Hi @jasonrandrews 
Thank you for the update and providing the repository link! I’ve successfully cloned the repository, and I appreciate the quick fix on the Learning Path. 
Thanks again for the quick response!

**Comment by nehcauhiy** [comment]
Hi @jasonrandrews
Thank you for the update and providing the repository link! I’ve successfully cloned the repository, and I appreciate the quick fix on the Learning Path.
Thanks again for the quick response!
